quantum/rgblight.c

```diff
@@ -25,6 +25,7 @@
 #include "rgblight.h"
 #include "debug.h"
 #include "led_tables.h"
+#include "velocikey.h"
```

Comment:
Actually, it may be better to use this:
```suggestion
#ifdef VELOCIKEY_ENABLE
  #include "velocikey.h"
#endif
```